docs/api/ShallowWrapper/prop.md

```diff
@@ -2,24 +2,21 @@
 
 Returns the prop value for the node of the current wrapper with the provided key.
 
-NOTE: can only be called on a wrapper of a single node.
+NOTE: can only be called on a wrapper of a single node. Cannot be called on the root wrapper because that represents what the component renders, not the component itself.
```

Comment:
It actually can be called on the root wrapper, but it will return the props on the rendered root element.